src/Patronage.Api/Controllers/ProjectController.cs

```diff
@@ -92,31 +122,54 @@ public ActionResult UpdateProject([FromRoute] int id, [FromBody] CreateOrUpdateP
         [HttpPatch("{id}")]
         [ProducesResponseType(StatusCodes.Status200OK)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
-        public ActionResult LightUpdateProject([FromRoute] int id, [FromBody] PartialProjectDto projectDto)
+        public async Task<ActionResult> LightUpdateProject([FromRoute] int id, [FromBody] PartialProjectDto projectDto)
         {
-            var isUpdated = _mediator.Send(new LightUpdateProjectCommand(id, projectDto));
-
-            //if (!isUpdated) return NotFound();
-
-            //return Ok();
-            return NoContent();
+            var isExistingRecord = await _mediator.Send(new LightUpdateProjectCommand(id, projectDto));
+
+            if (!isExistingRecord)
+            {
+                return NotFound(new BaseResponse<ProjectDto>
+                {
+                    ResponseCode = StatusCodes.Status404NotFound,
+                    Message = "Project with this id doesn't exist or you try to insert null value to property that can not be null or Name/Alias" +
+                    "has been used to another project (these properties must be unique)"
```

Comment:
```suggestion
                    Message = "Project with this id doesn't exist or you are trying to insert null value to property that is not nullable" +
                    "has been used to another project (these properties must be unique)"
```
Why are you concatenating strings?